alcs-frontend/src/app/features/notice-of-intent/documents/documents.component.scss

```diff
@@ -17,3 +17,20 @@
 a {
   word-break: break-all;
 }
+
+table {
+  position: relative; 
+
+  th mat-header-cell {
+    position: relative; 
+  }
+
+  .subheading {
+    font-size: 11px;
+    line-height: 16px;
```

Comment:
Should these be using the `rem` function?

Reply:
its kept common from alcs-frontend/src/app/features/application/decision/decision-v2/decision-documents/decision-documents.component.scss

Reply:
We use rem in the portal but px in ALCS. ALCS is not responsive whereas the Portal is.